diff --git a/src/client-community-detail-web.js b/src/client-community-detail-web.js
index 693ef26e..97e558f4 100644
--- a/src/client-community-detail-web.js
+++ b/src/client-community-detail-web.js
@@ -28,6 +28,7 @@ import AskAgentQuestionHowItWorksBannerNotificationContainer from 'sly/container
 import CommunityDetailsPageColumnContainer from 'sly/containers/CommunityDetailsPageColumnContainer';
 import RetentionPopup from 'sly/services/retentionPopup';
 import Image from 'sly/components/atoms/Image';
+import CommunityProfileAdTileContainer from 'sly/containers/communityProfile/AdTileContainer';
 
 const root = document.getElementById('app');
 
@@ -57,6 +58,7 @@ partiallyHydrateClient(
     CommunityDetailsPageColumnContainer,
     RetentionPopup,
     Image,
+    CommunityProfileAdTileContainer,
   ],
   clientConfigs.find(x => x.bundle === 'community-details').path,
   root,
diff --git a/src/components/pages/CommunityDetailPage/index.js b/src/components/pages/CommunityDetailPage/index.js
index 6a9fc9eb..8f69a86e 100644
--- a/src/components/pages/CommunityDetailPage/index.js
+++ b/src/components/pages/CommunityDetailPage/index.js
@@ -64,7 +64,7 @@ import { PROFILE_VIEWED } from 'sly/services/newApi/constants';
 import HeadingBoxSection from 'sly/components/molecules/HeadingBoxSection';
 import UnhydratedPageEventsContainer from 'sly/containers/PageEventsContainer';
 import UnhydratedCommunityDetailsPageColumnContainer from 'sly/containers/CommunityDetailsPageColumnContainer';
-// import UnhydratedCommunityProfileAdTileContainer from 'sly/containers/communityProfile/AdTileContainer';
+import UnhydratedCommunityProfileAdTileContainer from 'sly/containers/communityProfile/AdTileContainer';
 
 const PageViewActionContainer = withHydration(UnhydratedPageViewActionContainer, { alwaysHydrate: true });
 const PageEventsContainer = withHydration(UnhydratedPageEventsContainer, { alwaysHydrate: true });
@@ -84,7 +84,7 @@ const CommunityStickyFooter = withHydration(UnhydratedCommunityStickyFooter, { a
 const CommunityMorePicturesContainer = withHydration(UnhydratedCommunityMorePicturesContainer);
 const LazyCommunityMap = withHydration(UnhydratedLazyCommunityMap);
 const CommunityDetailsPageColumnContainer = withHydration(UnhydratedCommunityDetailsPageColumnContainer);
-// const CommunityProfileAdTileContainer = withHydration(UnhydratedCommunityProfileAdTileContainer, { alwaysHydrate: true });
+const CommunityProfileAdTileContainer = withHydration(UnhydratedCommunityProfileAdTileContainer, { alwaysHydrate: true });
 
 const BackToSearch = styled.div`
   text-align: center;
@@ -479,9 +479,9 @@ export default class CommunityDetailPage extends Component {
                   )}
                 </StyledHeadingBoxSection>
                 {/* TODO: ENABLE AFTER FIGURING OUT HYDRATION*/}
-                {/*<AdWrapper>*/}
-                  {/*<CommunityProfileAdTileContainer type="homeCare" community={community} />*/}
-                {/*</AdWrapper>*/}
+                <AdWrapper>
+                  <CommunityProfileAdTileContainer type="homeCare" community={community} />
+                </AdWrapper>
                 <StyledHeadingBoxSection
                   heading={`Get Availability at ${name}`}
                   id="availability"
